webapp/src/app/shared/widgets/character-widget/character-widget.component.html

```diff
@@ -0,0 +1,20 @@
+<div *ngIf="nested; else standAlone">
```

Comment:
If the nested block is not relevant for this pull request it should be removed. There is probably a better way then duplicating code for the other widget to work, but let's first try to merge this one in.